src/ipfs/publish_to_ipfs.sh

```diff
@@ -1,4 +1,11 @@
 #!/bin/bash
-
 source /app/.env
-/usr/local/bin/ipfs name publish `/usr/local/bin/ipfs add -Q -r /app/repository`
+export PATH=~$PATH:/usr/local/bin:/usr/local/openjdk-11/bin:/usr/bin
+export JAVA_HOME=/usr/local/openjdk-11/
+
+hash=$(ipfs add -Q -r /app/repository)
+echo $hash
+curl -X POST "https://ipfs.komputing.org/api/v0/pin/add?arg=$hash&recursive=true"
+ipfs name publish $hash
+
+/app/source_verify_ens_updater/bin/source_verify_ens_updater /app/repository
```

Comment:
Now you removed the IPFS secret here which is correct.
But the parameter still needs to be there - just the ENS key not the IPFS key. As Far as I see this way it would just fail.
Also ENS publishing should not be in a file called publish_to_ipfs.sh - this is not really separation of concerns and we (or other people) might stumble over this later

Reply:
So we would have two scripts one for updating ens and one ipfs uploader? And ens will run after ipfs is finished?

Reply:
yes. Another option would just to give it another (more generic) name like publisher. But I lean on just having 2 scripts. IMHO this is cleaner. Separation of concerns FTW.

Reply:
Agreed, will seperate in two.